utils/frame/benchmarking-cli/src/lib.rs

```diff
@@ -54,6 +54,10 @@ pub struct BenchmarkCmd {
 	#[structopt(short, long, default_value = "1")]
 	pub repeat: u32,
 
+	/// Select how many repetitions of this benchmark should run.
```

Comment:
is this totally disjoint from the previous `repeats`? as in, you can have 0 repeats and 10 internal repeats, but also 5 repeats and 5 internal repeats, and all combinations are valid?

Reply:
yes, they are totally disjoint. if you do 5 and 5, then you will do 25 iterations total